### DISCUSSION TITLE
remote: Permission to username/username.git denied to github-actions[bot].

### DISCUSSION THREAD

**Discussion by g0khul** [gautamkrishnar/blog-post-workflow#219] [Q&A] [answered]
I've just added this blog post workflow. Everything seems to work fine but my .git is denied access. How could I resolve it?
![Screenshot from 2023-05-17 20-28-53](https://github.com/gautamkrishnar/blog-post-workflow/assets/84320534/0fb73980-d84d-4172-b154-6be02664ca50)

**Comment by gautamkrishnar** [comment, marked as answer]
@igokhul please try https://github.com/gautamkrishnar/blog-post-workflow#faqs-and-common-issues

**Comment by g0khul** [comment]
Thank you I got it  displaying
